assets/wizards/setup/index.js

```diff
@@ -0,0 +1,275 @@
+	 */
+	completeSetup = () => {
+		const { setError } = this.props;
+		const params = {
+			path: `/newspack/v1/wizard/newspack-setup-wizard/complete`,
+			method: 'post',
+		};
+		return new Promise( ( resolve, reject ) => {
+			return apiFetch( params )
+				.then( response => {
+					this.setState( { setupComplete: true }, () => resolve( response ) );
+				} )
+				.catch( error => {
+					console.log( '[Complete Setup Error]', error );
+					setError( error );
+					reject( error );
+				} );
+		} );
+	};
+
+	isConfigurationComplete = () => {
+		const { pluginInfo } = this.state;
+		return (
+			pluginInfo[ 'jetpack' ] &&
+			pluginInfo[ 'jetpack' ].Configured &&
+			pluginInfo[ 'google-site-kit' ] &&
+			pluginInfo[ 'google-site-kit' ].Configured
+		);
+	};
+
+	getFirstPluginToConfigure = () => {
+		const { pluginInfo } = this.state;
+		if ( ! pluginInfo[ 'jetpack' ] || ! pluginInfo[ 'google-site-kit' ] ) {
+			return null;
+		}
+		if ( ! pluginInfo[ 'jetpack' ].Configured ) {
+			return { handoff: 'jetpack' };
+		}
+		if ( ! pluginInfo[ 'google-site-kit' ].Configured ) {
+			return { handoff: 'google-site-kit' };
+		}
+		return null;
+	};
+
+	/**
+	 * Render.
+	 */
+	render() {
+		const {
+			installedPlugins,
+			installationComplete,
+			pluginInfo,
+			profile,
+			countries,
+			currencies,
+			setupComplete,
+		} = this.state;
+		const installProgress = Object.keys( installedPlugins ).length;
+		const installTotal = REQUIRED_PLUGINS.length;
+		return (
+			<Fragment>
+				<NewspackLogo width="240" className="newspack-logo" />
+				<HashRouter hashType="slash">
+					<Switch>
+						<Route
+							path="/"
+							exact
+							render={ routeProps => (
+								<Welcome
+									buttonText={ __( 'Get started' ) }
+									buttonAction={ {
+										href: '#/about',
+										onClick: () => this.updateProfile(),
+									} }
+									profile={ profile }
+								/>
+							) }
+						/>
+						<Route
+							path="/about"
+							render={ routeProps => (
+								<About
+									headerText={ __( 'About your publication' ) }
+									subHeaderText={ __(
+										'Share a few details so we can start setting up your profile'
+									) }
+									buttonText={ __( 'Continue' ) }
+									buttonAction={ {
+										href: '#/newsroom',
+										onClick: () => this.updateProfile(),
+									} }
+									profile={ profile }
+									currencies={ currencies }
+									countries={ countries }
+									updateProfile={ ( key, value ) => {
+										this.setState( { profile: { ...profile, [ key ]: value } } );
+									} }
+								/>
+							) }
+						/>
+						<Route
+							path="/newsroom"
+							render={ routeProps => (
+								<Newsroom
+									headerText={ __( 'Tell us about your Newsroom' ) }
+									subHeaderText={ __(
+										'The description helps set the stage for the step content below'
+									) }
+									buttonText={ __( 'Continue' ) }
+									buttonAction="#/configure-plugins"
```

Comment:
This should also have `onClick: () => this.updateProfile()` like the address settings, right? Currently these settings don't save.

Reply:
Good eye, I probably never would have caught this! https://github.com/Automattic/newspack-plugin/pull/114/commits/a8e730eb198c3ba8ac3b36ed2855f4dfd221cc57